<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlterRunsWindow</class>
 <widget class="QDialog" name="AlterRunsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改班次</string>
  </property>
  <widget class="QLineEdit" name="capacityLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>218</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startStationLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>108</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>起点站：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endStationLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>138</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="departLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>68</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>发车时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="runsLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>18</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tripTimeLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>178</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="runsLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>28</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>班次：</string>
   </property>
  </widget>
  <widget class="QLabel" name="tripTimeLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>188</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>行车时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startStationLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>98</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ticketSoldLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>268</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>已售票：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ticketSoldLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>258</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="departTimeLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>58</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="capacityLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>228</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>额定载量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="endStationLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>148</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>终点站：</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>(HH:mm)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlterRunsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
